src/portfft/enums.hpp

```diff
@@ -43,6 +43,20 @@ enum class pad { DONT_PAD, DO_PAD };
 
 enum class level { WORKITEM, SUBGROUP, WORKGROUP, GLOBAL };
 
+constexpr const char* level_to_string(level l) {
```

Comment:
Are you using this somewhere?

Reply:
no, woops https://github.com/codeplaysoftware/portFFT/pull/136/commits/cdf802c3685a04dc5d9ed6a728ba873db0724280